amundsen_application/api/utils/metadata_utils.py

```diff
@@ -56,6 +57,19 @@ def marshall_table_full(table: Dict) -> Dict:
 
     results = {field: table.get(field, None) for field in fields}
 
+    # Temp code
+    if not results['badges']:
```

Comment:
remove?

Reply:
Yep, this is still a WIP. I'm currently writing some tests.